src/impl/util.js

```diff
@@ -208,18 +216,20 @@ export function signedOffset(offHourStr, offMinuteStr) {
 
 // COERCION
 
-export function normalizeObject(obj, normalizer, ignoreUnknown = false) {
+function asNumber(value) {
+  const numericValue = Number(value);
+  if (typeof value === "boolean" || value === "" || Number.isNaN(numericValue))
+    throw new Error(`Invalid unit value ${value}`);
```

Comment:
Auto comment here. This could be a Luxon Error subclass, such as `InvalidArgumentError`. But this would add a dependency to `errors.js` in what is today a leaf in the include dependencies.

Reply:
I hear you, but I think we should be consistent, even it creates some dependency cycles.

Reply:
I agree.